controllers/managedocs_controller.go

```diff
@@ -72,6 +72,7 @@ const (
 	storageClassSizeKey                    = "size"
 	enableMCGKey                           = "enable-mcg"
 	notificationEmailKeyPrefix             = "notification-email"
+	onboardingTicketKey                    = "onboarding-ticket-key"
```

Comment:
Why is the field name have `key` in it? (not the var name). 
Also, providers do not have onboarding tickets. 
I think what you are referring to is "onboarding-validation-key"